src/Dapr.Client/DaprClient.cs

```diff
@@ -143,6 +143,16 @@ public abstract class DaprClient
         /// <returns>A <see cref="ValueTask{T}" /> that will return the value when the operation has completed.</returns>
         public abstract ValueTask<TValue> GetStateAsync<TValue>(string storeName, string key, ConsistencyMode? consistencyMode = default, CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Gets a list of values associated with the <paramref name="keys" /> from the Dapr state store.
+        /// </summary>
+        /// <param name="storeName">The name of state store to read from.</param>
+        /// <param name="keys">The list of keys to get values for.</param>
+        /// <param name="parallelism">The number of concurrent get operations the Dapr runtime will issue to the state store. a value equal to or smaller than 0 means max parallelism.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
+        /// <returns>A <see cref="ValueTask{IReadOnlyList}" /> that will return the list of values when the operation has completed.</returns>
+        
+        public abstract ValueTask<IReadOnlyList<GetBulkItem>> GetBulkStateAsync(string storeName, IList<string> keys, int parallelism = -1, CancellationToken cancellationToken = default);
```

Comment:
Now this should be `IReadOnlyList<>` instead of `IList<>` 😆 - that makes it explicit that we're not going to mess with your parameter.

Reply:
Done